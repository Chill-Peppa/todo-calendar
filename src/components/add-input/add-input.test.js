import React from 'react';
import renderer from 'react-test-renderer';

import AddInput from './add-input';

const addNewTodo = jest.fn((todo) => {
  return {
    id: 1,
    todo: todo,
    isDone: false,
    date: '2022-01-01',
  };
});

const selectedDate = '2022-01-01';

it('should correctly render AddInput', () => {
  const tree = renderer
    .create(<AddInput addNewTodo={addNewTodo} selectedDate={selectedDate} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
