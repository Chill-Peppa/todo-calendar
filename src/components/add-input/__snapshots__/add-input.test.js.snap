// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render AddInput 1`] = `
<div
  className="modal__add-container"
>
  <input
    className="modal__input"
    id="add-input"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    placeholder="Добавьте новое событие"
    type="text"
    value=""
  />
  <button
    className="modal__add-button"
    onClick={[Function]}
    type="button"
  >
    +
  </button>
</div>
`;
